Automattic/jetpack
# Subscription Site: Polishing Newsletter settings

Part of https://github.com/Automattic/jetpack/issues/36021

See p1HpG7-rwo-p2

## Proposed changes:

It:
- reorders toggles, first Subscribe block and pop-up, then comments
- adds the "Preview and edit" link to the Subscribe block placement settings for block themes
- replaces the existing "Preview and edit the pop-up" link with the "Preview and edit" to keep the consistency
- puts the "subscription pop-up" toggle explanation to the dedicated `SupportInfo` block


### Before

<img width="1058" alt="Screenshot 2024-03-11 at 11 14 11" src="https://github.com/Automattic/jetpack/assets/4068554/32a20531-49a7-47bb-8b85-dd656dc23cd7">

### After

<img width="1058" alt="Screenshot 2024-03-11 at 11 13 48" src="https://github.com/Automattic/jetpack/assets/4068554/c8dc0974-7e40-4800-899a-5846611b019a">

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






- enable the Subscription Site feature by adding the following line to your `0-sandbox.php` file:
```php
add_filter( 'jetpack_subscription_site_enabled', '__return_true' );
```
- enable the Jetpack Newsletter Module
- go to the Newsletter settings and make sure the changes look sane
- with the block theme enabled make sure the"Preview and edit" links are visible and work as expected
- with the classic theme enabled make sure the "Preview and edit" links are not visible
